queries/rescript/highlights.scm

```diff
@@ -7,30 +7,58 @@
 ((value_identifier) @constant.macro
  (#match? @constant.macro "^\\.*$"))
 
+
+((value_identifier) @variable)
+
 [
   (type_identifier)
   (unit_type)
   (list)
   (list_pattern)
 ] @type
 
+((type_identifier) @type.builtin
+  (#any-of? @type.builtin
+    "int" "char" "string" "float" "bool" "unit"))
+
+
+((unit) @constant.builtin
+  (#set! "priority" 105))
```

Comment:
Usually in `nvim-treesitter` we try to avoid priority metadata, I think if you just place this query below the `@punctuation.bracket` query it should take precedence over it :+1:

Reply:
I tried that but I'm not sure why it doesn't work.